app/src/main/java/com/example/concordia_campus_guide/Fragments/POIFragment/POIFragmentViewModel.java

```diff
@@ -0,0 +1,6 @@
+package com.example.concordia_campus_guide.Fragments.POIFragment;
+
+import androidx.lifecycle.ViewModel;
+
+public class POIFragmentViewModel extends ViewModel {
```

Comment:
when you start implementing this class use the factory method to create an object of this class. to avoid using context. check up the info card fragment view model as a reference.

Reply:
Yes it will be used in the next PR for this task.

Reply:
ok I ll keep it in mind